src/libraries/Microsoft.NETCore.Platforms/src/Microsoft.NETCore.Platforms.csproj

```diff
@@ -58,16 +56,20 @@
   
   <UsingTask TaskName="GenerateRuntimeGraph" AssemblyFile="$(_generateRuntimeGraphTask)"/>
 
-  <Target Name="GenerateRuntimeJson" Condition="'$(AdditionalRuntimeIdentifiers)' != ''">
+  <Target Name="GenerateRuntimeJson" BeforeTargets="UpdateRuntimeJson">
     <MakeDir Directories="$(IntermediateOutputPath)" />
-    <GenerateRuntimeGraph RuntimeGroups="@(RuntimeGroupWithQualifiers)"
+    <GenerateRuntimeGraph Condition="'$(AdditionalRuntimeIdentifiers)' != ''"
+                          RuntimeGroups="@(RuntimeGroupWithQualifiers)"
                           AdditionalRuntimeIdentifiers="$(AdditionalRuntimeIdentifiers)"
                           AdditionalRuntimeIdentifierParent="$(AdditionalRuntimeIdentifierParent)"
-                          RuntimeJson="$(IntermediateOutputPath)runtime.json"
+                          RuntimeJson="$(GeneratedRuntimeIdentifierGraphFile)"
                           UpdateRuntimeFiles="True" />
+    <Copy Condition="'$(AdditionalRuntimeIdentifiers)' == ''"
+          SourceFiles="runtime.json"
+          DestinationFiles="$(GeneratedRuntimeIdentifierGraphFile)" />
   </Target>
 
-  <Target Name="UpdateRuntimeJson">
+  <Target Name="UpdateRuntimeJson" BeforeTargets="Build">
```

Comment:
By running this before build, won't it try to update the runtime.json in the source directory during the official builds?  This target was only meant to be manually invoked by the developer to update the checked in runtime.json.

Reply:
I've updated the Targets so only GenerateRuntimeJson is performed on Build.

Reply:
Didn't this target already run as part of the official build as it was already hooked onto `PackDependsOn`?